packages/homepage/src/pages/users.js

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+import { graphql } from 'gatsby';
+import styled from 'styled-components';
+import TitleAndMetaTags from '../components/TitleAndMetaTags';
+import PageContainer from '../components/PageContainer';
+import { Heading2 } from '../components/headings';
+import Layout from '../components/layout';
+import { Container } from './_explore.elements';
+import shuffleArray from '../utils/shuffleArray';
+
+const Grid = styled.section`
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(200px, 1fr));
+  grid-gap: 30px;
+  align-items: center;
+  margin-top: 60px;
+
+  img {
+    display: block;
+    max-width: 80%;
+    margin: auto;
+  }
+`;
+
+const Button = styled.a`
+  transition: 0.3s ease all;
+  color: white;
+  background-color: ${props => props.theme.secondary};
+  text-decoration: none;
+  padding: 0.5rem 12px;
+  text-align: center;
+  border-radius: 4px;
+  display: block;
+  width: 200px;
+
+  margin: auto;
+  margin-top: 1rem;
+  box-shadow: 0 2px 3px rgba(0, 0, 0, 0.3);
+
+  &:hover {
+    box-shadow: 0 7px 12px rgba(0, 0, 0, 0.3);
+  }
+`;
+
+export default ({
+  data: {
+    allUsersJson: { edges },
+  },
+}) => (
+  <Layout>
+    <Container>
+      <TitleAndMetaTags
+        description="Some of the people using codesandbox"
+        title="Who is using CodeSandbox? - CodeSandbox"
+      />
+      <PageContainer as="main" width={1440}>
+        <Heading2
+          css={`
+            text-align: center;
+          `}
+        >
+          Who's using CodeSandbox?
+        </Heading2>
+        <p
+          css={`
+            text-align: center;
+            color: white;
+          `}
+        >
+          Some amazing companies and projects using CodeSandbox
+        </p>
+        <Grid>
```

Comment:
Could we extract this component so we can reuse it on both pages?

Reply:
Created a `Companies.js` component